pkg/build/api/v1/defaults_test.go

```diff
@@ -148,7 +148,7 @@ func TestDefaults(t *testing.T) {
 }
 
 func roundTrip(t *testing.T, obj runtime.Object) runtime.Object {
-	data, err := runtime.Encode(kapi.Codecs.LegacyCodec(v1.SchemeGroupVersion), obj)
+	data, err := runtime.Encode(kapi.Codecs.LegacyCodec(v1.LegacySchemeGroupVersion), obj)
```

Comment:
actually I think we should (we can just make the the SGV a parameter to this function).